templates/primary-secondary/primary-secondary.js

```diff
@@ -902,15 +914,22 @@ class TemplatePrimarySecondary extends RtlMixin(LitElement) {
 
 		if (this._size === undefined) {
 			// initialize size on first resize
-			if (this._isMobile) {
-				this._size = this._contentBounds.minHeight;
+			const key = this._computeSizeKey();
+			const size = parseFloat(localStorage.getItem(key));
```

Comment:
You may want to test this in incognito mode in a few browsers... this used to throw in those scenarios, or if the locale storage was "full".